version/version.go

```diff
@@ -23,6 +23,9 @@ func GetVersion() *VersionInfo {
 	if GitDescribe != "" {
 		ver = GitDescribe
 	}
+	if GitDescribe == "" && rel == "" && VersionPrerelease != "" {
```

Comment:
This change shouldn't be reintroduced. When going through this file as part of introducing the `sys/internal/ui/version` path, I discovered that it was impossible for the condition on line 26 to be **true**. Because it requires **VersionPrerelease** to be not empty and **rel** to be empty. However, **rel** is initialized with the **VersionPrerelease** value above, so it would require **rel** to be a Schrödinger variable in order for the condition to be **true**.